check-submodules.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Checks out if a submodule has been updated
+SUBMODULES=`git submodule--helper list | awk '{ print $4 }'`
+RUN_CI="no"
+for SUBMODULE in $SUBMODULES
+do
+    git diff --exit-code HEAD^ $SUBMODULE
+    CHANGED=$? # Get git diff exit code
+    if [ $CHANGED -eq 1 ]
+    then
+        RUN_CI="yes"
+        SUBMODULE_NAME=$submodule
+        break
+    fi
+done
+
+# If no submodule has been updated, exit the script
+if [ "$RUN_CI" = "no" ]; then
+    exit 0
+fi
+
+# Install json-diff
+JSON_DIFF_LINK="https://github.com/Luni-4/json-diff/releases/download"
+JSON_DIFF_VERSION="0.0.1"
+curl -L "$JSON_DIFF_LINK/v$JSON_DIFF_VERSION/json-diff-linux.tar.gz" |
+tar xz -C $CARGO_HOME/bin
+
+# Install json minimal tests
+JMT_LINK="https://github.com/Luni-4/json-minimal-tests/releases/download"
```

Comment:
Same here

Reply:
This one won't be published. It's useless for any other user except us. Let's treat it with the { path: ...} notation if we want to add it to a future test

Reply:
It's OK, publishing doesn't hurt, indeed there are a lot of packages that are used by very few people. You never know, maybe it'll be useful for other projects of ours.
Anyway, it's a `nit`, so feel free to ignore ;)

Reply:
Updated on crates.io